lib/react_on_rails/helper.rb

```diff
@@ -109,8 +108,10 @@ def load_pack_for_component(component_name)
       append_stylesheet_pack_tag "generated/#{component_name}"
     end
 
-    def generated_components_pack(component_name)
-      "#{ReactOnRails::WebpackerUtils.webpacker_source_entry_path}/generated/#{component_name}"
+    def generated_components_pack_path(component_name)
```

Comment:
Should this be part of the helper? or not?

Reply:
It should be a private member